src/graphql/types/object/wallet-contact.ts

```diff
@@ -9,11 +9,12 @@ import * as Accounts from "@app/accounts"
 import Username from "../scalar/username"
 import { checkedToUsername } from "@domain/users"
 
+//FIXME: Discrepancy with filename
 const UserContact = new GT.Object({
   name: "UserContact",
   fields: () => ({
-    username: { type: GT.NonNull(Username) },
-    alias: { type: ContactAlias },
+    username: { type: GT.NonNull(Username), description: "Actual identifier of the contact." },
+    alias: { type: ContactAlias, description: "Alias the user can set for this contact.\nOnly the user can see the alias attached to their contact." },
```

Comment:
This line is a bit long - shouldn't get by linting or?
Isn't there a better way to do multi-line string that respects our line width settings?

Reply:
good idea. use: ``

Reply:
👍